ql/src/test/results/clientpositive/llap/limit_pushdown.q.out

```diff
@@ -1075,6 +1072,13 @@ STAGE PLANS:
                           Map-reduce partition columns: _col0 (type: string)
                           Statistics: Num rows: 316 Data size: 30020 Basic stats: COMPLETE Column stats: COMPLETE
                           value expressions: _col1 (type: bigint)
+            Execution mode: vectorized, llap
+            LLAP IO: all inputs
+        Map 3 
+            Map Operator Tree:
+                TableScan
+                  alias: src
+                  Statistics: Num rows: 500 Data size: 43500 Basic stats: COMPLETE Column stats: COMPLETE
                   Top N Key Operator
```

Comment:
in the old plan: did we have 2 TopN key operators in this plan which are equal?
this is unrelated to this patch; but we may have an issue with its comparision - and because of that SWO is not able to simplify them